src/pages/Cancel/Cancel.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+
+import { CancelWidget } from "../../components/CancelWidget/CancelWidget";
+import Page from "../../components/Page/Page";
+
+const Cancel: FC = () => {
+  return (
+    <Page>
+      <CancelWidget />
+    </Page>
+  );
+};
```

Comment:
If you land directly on a cancel page activeOrder would be undefined.

```suggestion
const Cancel: FC = () => {
  const dispatch = useAppDispatch();
  const { compressedOrder } = useParams<{ compressedOrder: string }>();

  const { status, activeOrder } = useAppSelector(selectTakeOtcReducer);

  useEffect(() => {
    if (compressedOrder && !activeOrder) {
      dispatch(decompressAndSetActiveOrder({ compressedOrder, tokens: [] }));
    }
  }, [dispatch, compressedOrder]);

  if (status === "idle" || !activeOrder) {
    return <Page />;
  }

  if (status === "not-found") {
    return (
      <Page>
        <InvalidOrder />
      </Page>
    );
  }

  return (
    <Page>
      <CancelWidget />
    </Page>
  );
};
```